components/mosart/src/riverroute/RtmIO.F90

```diff
@@ -1095,8 +1097,8 @@ subroutine ncd_io_int_var1_nf(varname, data, flag, ncid, readvar, nt)
        else
           start(1) = 1
           count(1) = size(data)
-          start(2) = 1
-          count(2) = 1
+          !start(2) = 1
```

Comment:
How about removing these entries instead of leaving them as comments?

Reply:
Ok, I will remove the commented code here and rebase the branch.